guide/additional-info/async-await.md

```diff
@@ -35,11 +35,28 @@ deleteMessages(5).then(value => {
 });
 ```
 
-In this scenario the `deleteMessages` function returns a Promise. The `.then()` function will trigger if the Promise was resolved and the `.catch()` function if the Promise was rejected. But with our function we resolve the Promise after 2 seconds with the String "Everything is fine", so the `.catch()` function will never be executed.
+In this scenario the `deleteMessages` function returns a Promise. The `.then()` method will trigger if the Promise was resolved and the `.catch()` method if the Promise was rejected. But with our function we resolve the Promise after 2 seconds with the String "Everything is fine", so the `.catch()` method will never be executed.
 
 ## How to implement async/await
 
-After knowing how Promises works and what they are for, let's look at an example in which we handle multiple Promises. Let's say you wanted to react to a message in a certain order with letters (regional indicators). For this example, I will take the basic template for a discord.js bot with some ES6 adjustments.
+### Theory information
+The following Information are important to know before starting working with async/await. You can only use the await keyword inside a function what is marked as async (you call the async keyword before the function name, function keyword or the parameter when you use an arrow function) a simple example would be 
+```js
+async function markedAsAsync() {
+	// code
+}
+``` 
+or 
+```js 
+async () => {
+	// code
+}
+```
+also an important thing to know is that a function marked as async will always return a Promise. In addition to this if you return something inside a async function this value will be the resolved value and if you throw an error inside an async function it will reject this Promise with the error object. 
```

Comment:
- maybe capitalize `also`?
- I feel like there should be a comma after `to know`, this could be an optional or even wrong one though
- ^ for `In addition to this`, however this is an optional one (Just a personal opinion, you can let that out)
- `inside a_n_ async function`
- `if you return something ... the (returned) promise will resolve with the returned value`. `returned` feels a bit redundant, not sure whether this is actually needed.
- `if you throw an error ... it will reject _the_ Promise with the _thrown_ error.` ~~(You can technically throw anything although it should be an Error)~~
- maybe also a comma after `throw an error inside an async function`?

Reply:
because this guide shows best practise i would only covering throwing Errors inside a asnyc function because you should always reject with an actuall error object

Reply:
Correct, thus strikethrough :P